rustls/src/conn.rs

```diff
@@ -635,7 +635,12 @@ impl<Data> ConnectionCore<Data> {
             }
         };
 
-        while let Some(msg) = self.deframe()? {
```

Comment:
I think the formatting makes this pretty hard to follow. Maybe unroll the `while` loop into `loop { let msg = match self.deframe() { .. }; .. }` instead?

Reply:
I suggest we just put this in `fn deframe` a few lines below. It already has a specific match arm for `Error::DecryptError`.

Reply:
right, that looks better like this: bc09dbf8